.github/workflows/main.yml

```diff
@@ -92,11 +92,11 @@ jobs:
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
           onlyChanged: "!startsWith(github.ref, 'refs/heads/main')" # Use TurboSnap for PR builds
           externals: |
-            - './packages/nimble-components/.storybook/public/**'
+            - './packages/storybook/.storybook/public/**'
             - './packages/nimble-tokens/dist/icons/svg/**'
```

Comment:
Didn't think it through and had [merged a storybook lockfile update in main](https://github.com/ni/nimble/pull/2049) but decided to revert it to prevent conflicts with this PR. Shouldn't have a net impact here but an FYI.